src/testing/testing.go

```diff
@@ -280,8 +280,9 @@ func (c *common) Skipf(format string, args ...interface{}) {
 // by calling runtime.Goexit.
 func (c *common) SkipNow() {
 	c.skip()
+	c.skipped = true
```

Comment:
The call to `c.skip()` already does  `c.skipped = true` so this line is redundant.

Reply:
done @deadprogram